modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/liferay-clay.tld

```diff
@@ -1513,6 +1513,12 @@
 			<required>false</required>
 			<rtexprvalue>true</rtexprvalue>
 		</attribute>
+		<attribute>
+			<description></description>
+			<name>disableIncompatibleActions</name>
+			<required>false</required>
+			<rtexprvalue>true</rtexprvalue>
+		</attribute>
```

Comment:
Do we need this? Isn't just `multipleTypesBulkActionDisabled` enough?

Reply:
It's leftover from the previous iteration 👀